ui/src/views/404/NavigationItem/style.js.scss

```diff
@@ -0,0 +1,30 @@
+nav_item {
+    display: block;
+    padding: 10px 2.5%;
+    margin: 0 2.5%;
+    border-bottom: 1px solid #f4f4ee;
```

Comment:
agreed - these colours are the current ones on the 404, which are outdated, hence not wanting to appear to 'formalise' them the past file.

we could use proper ones, in which case i totally agree they should come out of pasteup (along with font sizes, families etc)